schema/omega-services.async.api.yaml

```diff
@@ -414,6 +545,222 @@ components:
           items:
             $ref: '#/components/schemas/GeneralDatedDescription'
 
+    ListAgentParameters:
+      type: object
+      properties:
+        type:
+          $ref: '#/components/schemas/AgentTypes'
+          description: Limits the list to one-or-more types of Agent. If omitted, all types of agents are included in the list.
+        authority-file:
+          $ref: '#/components/schemas/AuthorityFile'
+          description: Limits the list of Agent(s) to only those that are (or are not) also Authority Files. If omitted both authorative and non-authorative Agents are included in the list.
+        depository:
+          $ref: '#/components/schemas/Depository'
+          description: Limits the list of Agent(s) to only those that are (or are not) also a depository. If omitted both depository and non-depository Agents are included in the list.
+        version-timestamp:
+          $ref: '#/components/schemas/VersionTimestamp'
+          description: Limits the list of Agent(s) to only those that have a version timestamp that is greated than or equal to this timestamp.
+          default: latest
+
+    AgentSummary:
+      type: object
+      required:
+        - identifier
+        - type
+        - current-description
+        - description
+      properties:
+        identifier:
+          $ref: '#/components/schemas/GeneralIdentifier'
+        type:
+          $ref: '#/components/schemas/AgentType'
+        current-description:
+          $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the current description of the Agent concept
+        description:
+          type: array
+          items:
+            $ref: '#/components/schemas/AgentDescriptionSummary'
```

Comment:
When you take into account the `AgentDescriptionSummary` there is a lot more information in here than I expected. Currently we are only interested in the `identifier` and `label` so I imagined something more like the `LegalStatusSummary`.

Also I notice the `AgentDescriptionSummary` is an array. Can I assume if I request `latest` (the default) there will only be one description?

Reply:
@adamretter, in the current UI these fields are concatenated and the text is displayed in a drop-down. In the new UI, which you can see in the prototype here, the name and date fields are separated and so two fields would be more ideal, one with a label for name and optional title and another with the date range.

Reply:
Thanks Rob, I understand now. I have appended a commit to include the `date-from` and `date-to` properties in the Summary.